openerp/addons/base/ir/ir_attachment.py

```diff
@@ -427,8 +430,26 @@ def create(self, cr, uid, values, context=None):
             values.pop(field, False)
         values = self._check_contents(cr, uid, values, context=context)
         self.check(cr, uid, [], mode='write', context=context, values=values)
+        if 'url' in values:
```

Comment:
I also don't think so for the standard code, but I think this must anticipate whatever some other code does. So if there's some module that creates new assets without deleting the old ones, it would break without this, that's why I'd prefer to leave this in here